website/aws.erb

```diff
@@ -403,6 +403,15 @@
                     </ul>
                 </li>
 
+                <li<%= sidebar_current("docs-aws-resource-budget") %>>
+                    <a href="#">Budget Resources</a>
+                    <ul class="nav nav-visible">
+                        <li<%= sidebar_current("docs-aws-resource-budget") %>>
+                            <a href="/docs/providers/aws/r/budget.html">aws_budget</a>
```

Comment:
The file path and resource name need to be updated for the nesting `<a href="/docs/providers/aws/r/budgets_budget.html">aws_budgets_budget</a>`